lang/luasocket/Makefile

```diff
@@ -8,33 +8,59 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=luasocket
-PKG_SOURCE_VERSION:=6d5e40c324c84d9c1453ae88e0ad5bdd0a631448
-PKG_VERSION:=3.0-rc1-20130909
-PKG_RELEASE:=5
+PKG_SOURCE_DATE:=2019-04-21
```

Comment:
I've got to say, bumping verisons _realllllly_ should be separate from "providing support for lua5.3"!

Reply:
@karlp - to support 5.3 I needed to upgrade the source.  How would you like me to proceed?

Reply:
Two different commits. First one bumping `PKG_SOURCE_DATE` (and updating `PKG_RELEASE`). Later one adding 5.3.

Reply:
@rmilecki, @karlp, I've split the single commit into two commits.  I believe this is what you requested.